db/schema.rb

```diff
@@ -51,9 +51,23 @@
     t.integer  "application_id",   limit: 4
     t.string   "trailing_message", limit: 255
     t.integer  "event_id",         limit: 4
+    t.integer  "ip_address",       limit: 8
+    t.integer  "user_agent_id",    limit: 4
   end
 
   add_index "event_logs", ["uid", "created_at"], name: "index_event_logs_on_uid_and_created_at", using: :btree
+  add_index "event_logs", ["user_agent_id"], name: "event_logs_user_agent_id_fk", using: :btree
+
+  create_table "lhma_2017_09_11_10_42_28_471_event_logs", force: :cascade do |t|
+    t.string   "uid",              limit: 255, null: false
+    t.datetime "created_at",                   null: false
+    t.integer  "initiator_id",     limit: 4
+    t.integer  "application_id",   limit: 4
+    t.string   "trailing_message", limit: 255
+    t.integer  "event_id",         limit: 4
+  end
+
+  add_index "lhma_2017_09_11_10_42_28_471_event_logs", ["uid", "created_at"], name: "lhm_index_event_logs_on_uid_and_created_at", using: :btree
```

Comment:
We shouldn't be adding this to the schema.  My guess is that you've run this migration locally and rails has picked it up from your dev DB when it generated the schema.  It's not unusual for a dev DB to have a few local changes that we don't want to commit, so we need to be careful when adding `schema.rb` to git to make sure that we don't add more than we expect.